### ISSUE TITLE
Multiple "timed out" logs from pipestream.go

### ISSUE THREAD

**Issue by mveroone** [google/mtail#462]
Hey.

Follow-up on #339 since you reworked the named pipe reading mecanism.
It works great now, although the logs are filled with this kind of message :
```
I0203 17:00:56.575733   21953 pipestream.go:95] timed out
...
```
8 lines in a few milliseconds, repeating every 500ms.
Which makes the log file 500MB big after a week or so of runtime.

Note that this particular instance of mtail is reading from 25 different pipes(from which 8 are very active and the other 17 much less so) from syslog-ng, for about 60 million log lines a day.
I believe this could be related to the way syslog-ng writes into named pipes ?

We're running version 3.0.0-rc42 on an up to date RHEL 7
Command-line flags : `--disable_fsnotify --poll_interval 500ms`

**Comment by jaqx0r** [comment]
Yow.  That's an expected condition so the log should be downgraded to V2.
I'll send a fix.

**Comment by jaqx0r** [comment]
BTW glad to hear the fix for #339 is good!

**Comment by mveroone** [comment]
Thanks ! 🙏
